src/node_metadata.cc

```diff
@@ -8,6 +8,7 @@
 #include "uv.h"
 #include "v8.h"
 #include "zlib.h"
+#include "../deps/uvwasi/include/uvwasi.h"
```

Comment:
The WASI implementation is able to use `#include "uvwasi.h"`. Does that not work here?

Reply:
I missed it.

Reply:
Added thanks.